tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Extensions/ClaimsPrincipalExtensionsTests.cs

```diff
@@ -73,5 +74,49 @@ internal static class ClaimsPrincipalExtensionsTests
 
             user.GetUserName().Should().Be(name);
         }
+
+        [Test]
+        [AutoData]
+        public static void IsAuthorisedForOrganisation_OrganisationIdIsPrimary_ReturnsTrue(            string name)
```

Comment:
Looks like we've got some superfluous white space 😃 

I'm surprised there wasn't a linting error.

Reply:
corrected